MESSAGE.md

```diff
@@ -3,6 +3,32 @@ OMMX Message
 
 [![doc](https://img.shields.io/badge/Protocol-Documentation-blue)](https://jij-inc.github.io/ommx/protobuf.html)
 
+## Why OMMX Message based on Protocol Buffers? Why not [JSON](https://www.json.org/json-en.html), [CBOR](https://cbor.io/), or [HDF5](https://www.hdfgroup.org/solutions/hdf5/)?
+
+:zap: We need to define a data schema for messages exchanged between applications, services, and databases that make up OROps.
+
+We have to discuss the following points to answer the question.
+
+- Where OMMX is used?
+- Why schema is required?
+- Text-based or Binary-based?
+- Why Protocol Buffers?
+
+Note that this is a **better** selection, not the **best** selection. We have to keep considering these points and change the selection if necessary.
+
+### Where OMMX is used?
+
+Mathematical programming has been studied long time in academic fields, and many software tools have been developed to improve the solving capability. However, these tools are basically designed to use in a research process. When we try to use these invaluable tools in a real-world business process, we call it "OROps" (OR = operations research) like "MLOps" in machine-learning field, we have to face many problems. The most significant problem is the lack of interoperability between tools. We have to convert data between tools, and it is a time-consuming and error-prone process. OMMX is designed to solve this problem.
```

Comment:
It's a matter of preference, and there are no significant changes to the content itself:

```suggestion
Mathematical programming has been a subject of study in academic fields for a considerable period of time, and numerous software tools have been developed to improve the solving capability. However, these tools are basically designed for use in a research process. When we attempt to use these invaluable tools in a real-world business process, we refer to it as "OROps" (OR = operations research), like "MLOps" in machine-learning field, we have to face many problems. The most significant problem is the lack of interoperability between tools. We have to convert data between tools, and it is a time-consuming and error-prone process. OMMX is designed to solve this problem.
```